pynetbox/core/response.py

```diff
@@ -58,9 +58,7 @@ def get_return(lookup, return_fields=None):
 
 
 def flatten_custom(custom_dict):
-    return {
-        k: v if not isinstance(v, dict) else v["value"] for k, v in custom_dict.items()
-    }
+    return {k: v for k, v in custom_dict.items()}
```

Comment:
This really seems like a NOOP to me. Perhaps it would be better to eliminate calls to flatten_custom altogether?

Reply:
@srfwx  weighed in on #457 back in June providing reasoning that `flatten_custom` is needed, and it looks like some discussion around this is the main thing preventing merge of an appropriate fix.